### PR TITLE
chore: adding node18 to CI config

### PR DESCRIPTION
**Pull request by marcelo-sousa-mh**
The release of the last commit [failed](https://github.com/latitudesh/latitudesh-nodejs/actions/runs/4105585289/jobs/7082667157) due to the Node version required by Semantic Release. According to [this doc](https://github.com/semantic-release/semantic-release/blob/master/docs/support/node-version.md), we need to run Node 18 on integration. 
I've added `18.x` under strategy, along with 12.x and 14.x. 
Since I'm not sure this is a safe move, I'm putting it up to review to learn what you guys think

### CODE REVIEW

**Review by LeoAltissimo — APPROVED** [review]
I see no problems with adding a version in the array. Following the documentation, adding an item does not invalidate the others. They will run in parallel.

https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,59 +2,57 @@ name: CI
...
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
```
Comment (maxweldsouza) [inline .github/workflows/ci.yml:49]: @marcelo-sousa-mh, I think this is the version that needs to be bumped.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -2,59 +2,57 @@ name: CI
...
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [12.x, 14.x, 18.x]
```
Comment (maxweldsouza) [inline .github/workflows/ci.yml:16]: I think we can drop 12.x because it has reached the end of life. We can add 16.x instead.

**Comment by marcelo-sousa-mh** [comment]
@maxweldsouza @LeoAltissimo Thanks guys, let's merge and see if it fixes!

**Pull request merged**